code/game/objects/items.dm

```diff
@@ -213,6 +213,18 @@
 	..()
 	qdel(src)
 
+var/global/objects_thrown_when_explode = FALSE
+
+/obj/item/throw_impact(atom/impacted_atom, speed, mob/user)
+	..()
+	if(isturf(loc))
+		var/turf/T = loc
+		if(objects_thrown_when_explode || T.arcanetampered)
+			if(T)
```

Comment:
This check doubly cannot possibly fail for not one, but two different reasons

Reply:
fixed